config/crd/v1/core.giantswarm.io_configs.yaml

```
@@ -0,0 +1,130 @@
+
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.2.4
+  creationTimestamp: null
+  name: configs.core.giantswarm.io
+spec:
+  group: core.giantswarm.io
+  names:
+    categories:
+    - config
+    - giantswarm
+    kind: Config
+    listKind: ConfigList
+    plural: configs
+    singular: config
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        description: Config represents configuration of a Management Cluster App.
```

```suggestion
        description: Config represents configuration of an App.
```

i think it's a bit cleaner if we just elide the cluster type here

---

Suggestions in `.go` files next time please, those files are generated :)

---

how do computers work